lib/ansible/plugins/inventory/scaleway.py

```diff
@@ -113,7 +113,7 @@ def _fetch_information(token, url):
         except KeyError:
             raise AnsibleError("Incorrect format from the Scaleway API response")
 
-        link = response.getheader('Link')
+        link = response.headers['Link']
```

Comment:
This is a difference between py2/py3.  Your change will work fine in both, and it still supports case insensitive key lookups.

Just for educational purposes, the `getheader` method exists at the following locations in py2/py3:

py2
```
response.headers.getheader
```

py3
```
response.getheader
```